tools/witx/tests/wasi.rs

```diff
@@ -155,7 +155,9 @@ fn diff_against_filesystem(expected: &str, path: &Path) {
     }
 
     eprintln!();
-    eprintln!("To regenerate the files, run `cd tools/witx && cargo run -- repo-docs`.");
+    eprintln!(
+        "To regenerate the files, run `cd tools/witx && cargo run -- --example witx repo-docs`."
```

Comment:
I think the `--` will need to be removed, but otherwise looks good!

Reply:
thanks!